elyra/pipeline/airflow/component_parser_airflow.py

```diff
@@ -127,6 +127,10 @@ def _parse_properties(self, content: Dict[str, List]) -> List[ComponentParameter
             description = self._format_description(description=description, data_type=data_type)
 
             data_type_info = self.determine_type_information(data_type)
+
+            if 'xcom' in arg.lower() and data_type_info.data_type == 'boolean':
+                data_type_info.default_value = True
+
             if data_type_info.undetermined:
                 self.log.debug(f"Data type from parsed data ('{data_type}') could not be determined. "
                                f"Proceeding as if 'string' was detected.")
```

Comment:
This is mostly a nit - so up to you, but I did a double-take that the `default_value = True` could be applied to an `undetermined` type and had to go look at `determine_type_information()` to find that isn't the case.  I would suggest swapping the two statements and not assume that `undetermined` types will always be strings (although, so long as they remain a usable component, 'string' is the right choice - this is more about visibility).  Perhaps something like the following?
```suggestion
            data_type_info = self.determine_type_information(data_type)
            if data_type_info.undetermined:
                self.log.debug(f"Data type from parsed data ('{data_type}') could not be determined. "
                               f"Proceeding as if '{data_type_info.data_type}' was detected.")
            elif 'xcom' in arg.lower() and data_type_info.data_type == 'boolean':
                data_type_info.default_value = True
```

Reply:
thx!!! committed